GDJS/Runtime/pixi-renderers/layer-pixi-renderer.js

```diff
@@ -54,34 +62,44 @@ gdjs.LayerPixiRenderer.prototype.updatePosition = function() {
   this._pixiContainer.position.y += this._layer.getHeight() / 2;
 };
 
-gdjs.LayerPixiRenderer.prototype.updateVisibility = function(visible) {
+gdjs.LayerPixiRenderer.prototype.updateVisibility = function (visible) {
   this._pixiContainer.visible = !!visible;
 };
 
-gdjs.LayerPixiRenderer.prototype.updateTime = function() {
-  for(var filterName in this._filters) {
+gdjs.LayerPixiRenderer.prototype.updateTime = function () {
```

Comment:
If this is doing something else than updating the time, we should probably rename this to `update` (also in the Cocos files)